tests/conftest.py

```diff
@@ -14,11 +14,9 @@
 
 
 def _read_prompts(filename: str) -> str:
```

Comment:
Looks like this function has wrong signature. output seems to be List[str]

Reply:
Good catch! fixed.